app/views/govuk_component/button.raw.html.erb

```diff
@@ -5,13 +5,15 @@
   rel ||= false
   text ||= ""
   tag_name = href ? "a" : "button"
+  margin_bottom ||= false
 %>
 <<%= tag_name %>
```

Comment:
What's going on here with the double <? Is it some magic syntax I'm not aware of, or a mistake?

Reply:
There's no changes to this in this PR. This allows us to make this 'button' into an anchor or a button depending on what is passed to it.